simpa/core/device_digital_twins/detection_geometries/ithera_invision_array.py

```diff
@@ -0,0 +1,353 @@
+# SPDX-FileCopyrightText: 2021 Computer Assisted Medical Interventions Group, DKFZ
+# SPDX-FileCopyrightText: 2021 Janek Groehl
+# SPDX-License-Identifier: MIT
+import numpy as np
+
+from simpa.core.device_digital_twins import DetectionGeometryBase
+from simpa.utils import Tags
+
+
+class iTheraInvision256TFDetectionGeometry(DetectionGeometryBase):
+    """
+    This class represents a digital twin of a ultrasound detection device
+    with a curved detection geometry. The origin for this device is the center (focus) of the curved array.
+    """
+
+    def __init__(self,
+                 device_position_mm=None,
+                 field_of_view_extent_mm=None):
+        """
+
+        :param pitch_mm: In-plane distance between the beginning of one detector element to the next detector element.
+        :param radius_mm:
+        :param number_detector_elements:
+        :param detector_element_width_mm:
+        :param detector_element_length_mm:
+        :param center_frequency_hz:
+        :param bandwidth_percent:
+        :param sampling_frequency_mhz:
+        :param angular_origin_offset:
+        :param device_position_mm: Center (focus) of the curved array.
+        """
+
+        super(iTheraInvision256TFDetectionGeometry, self).__init__(
+             number_detector_elements=256,
+             detector_element_width_mm=0.635,
+             detector_element_length_mm=15,
+             center_frequency_hz=5e6,
+             bandwidth_percent=55,
+             sampling_frequency_mhz=40,
+             device_position_mm=device_position_mm)
+
+        self.positions = (np.asarray([[0.02890019, -0.02837304, 0.],
```

Comment:
Would it be possible to calculate these positions based on some basic parameters or do you specifically want these positions and we are allowed by the vendor to publish these positions?